tests/test_sota_openvino.py

```diff
@@ -0,0 +1,148 @@
+                text('Model')
+            with tag('td'):
+                text('NCCF')
+            with tag('td'):
+                text('Accuracy Checker')
+            with tag('td'):
+                text('different')
+        for i in nncf_metrics:
+            with tag('tr'):
+                with tag('td'):
+                    text(i)
+                with tag('td'):
+                    text(nncf_metrics[i])
+                with tag('td'):
+                    text(ov_metrics[i])
+                with tag('td'):
+                    text(round((nncf_metrics[i] - ov_metrics[i]), 2))
+    f = open('Common_results.html', 'w')
+    f.write(doc.getvalue())
+    f.close()
+
+
+def onnx_model_name(name):
+    onnx_name = os.path.join(PROJECT_ROOT, "onnx", str(name+".onnx"))
+    return onnx_name
+
+
+def open_metrics_file():
+    with open("metrics.json") as metric_file:
+        metrics = json.load(metric_file)
+    return metrics
+
+
+def test_convert_to_onnx(sota_checkpoints_dir, sota_data_dir):
+    subprocess.run("pytest tests/test_sota_checkpoints.py --sota-checkpoints-dir={} --sota-data-dir={} -k test_eval"
+                   " --convert-to-onnx".format(sota_checkpoints_dir, sota_data_dir), check=True,
+                   shell=True)
+
+
+def test_eval_checkpoints(sota_checkpoints_dir, sota_data_dir):
+    subprocess.run("pytest tests/test_sota_checkpoints.py --sota-checkpoints-dir={} --sota-data-dir={} -k test_eval"
+                   .format(sota_checkpoints_dir, sota_data_dir), check=True, shell=True)
+
+
+def test_openvino_install(openvino_url):
+    subprocess.run("curl -o openvino.tgz {}".format(openvino_url), check=True, shell=True)
+    subprocess.run("tar -xvzf openvino.tgz", check=True, shell=True)
+    subprocess.run("sudo ./install.sh --silent", cwd=os.path.join(PROJECT_ROOT, "openvino"), check=True, shell=True)
+    subprocess.run("sudo ./install_openvino_dependencies.sh", cwd=os.path.join(OPENVINO_DIR, "install_dependencies"),
+                   check=True, shell=True)
+    subprocess.run("pip install -r requirements.txt", cwd=os.path.join(OPENVINO_DIR, "python"), check=True, shell=True)
+    subprocess.run("{} setup.py install".format(sys.executable), cwd=ACC_CHECK_DIR, check=True, shell=True)
+
+
+for k in open_metrics_file():
+    model_list.append(k)
+
+
+@pytest.mark.parametrize("model", model_list, ids=model_list)
+def test_convert_to_ir(model):
+    onnx_model = onnx_model_name(model)
+    subprocess.run("{} {} mo.py --input_model {} --framework=onnx --data_type=FP16 --reverse_input_channels"
+                   " --mean_values=[123.675,116.28,103.53] --scale_values=[58.4795,57.1429,57.4713]"
+                   " --output_dir {}".format(OPENVINO_ACTIVATE, sys.executable, onnx_model,
+                                             os.path.join(PROJECT_ROOT, "ir_models")),
+                   cwd=(os.path.join(OPENVINO_DIR, "deployment_tools", "model_optimizer")), check=True, shell=True)
```

Comment:
You separated different stages of one and the same test case into separate test cases, which is not necessarily wrong, but your IR conversion "test case" in fact should not be even run on its own without ONNX conversion, because it will definitely fail, and your `accuracy_checker` "test case" also cannot be run separately from the ONNX conversion and IR conversion. Either move all the stages into a single test case, or set up proper dependencies between the stages:
https://pytest-dependency.readthedocs.io/en/stable/usage.html